source/gui/settingsDialogs.py

```diff
@@ -3718,7 +3718,6 @@ def makeSettings(self, settingsSizer):
 		self.updateDriverSettings()
 
 		sHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
-

```

Comment:
this new line should be preserved

Reply:
this is unresolved somewhat - this should be an empty new line, not one with indentation